caluma/caluma_workflow/models.py

```diff
@@ -236,3 +249,16 @@ class Meta:
             GinIndex(fields=["assigned_users"]),
             GinIndex(fields=["meta"]),
         ]
+
+
+@receiver(post_init, sender=WorkItem)
+def set_name_and_description(sender, instance, **kwargs):
+    """
+    Ensure WorkItem has a name and description set.
+
+    Default to values from Task.
+    """
+    if not any((value for _, value in instance.name.items())):
+        instance.name = instance.task.name
+    if not any((value for _, value in instance.description.items())):
+        instance.description = instance.task.description
```

Comment:
Is it guaranteed that a work item always has a task, even at `post_init`?

Reply:
Yes, you can't save a `WorkItem` without a `Task`.

Reply:
Right, but a WorkItem can be created step-by-step, which would lead to a crash:
```python
>>> from caluma.caluma_workflow.models import *
>>> wi = WorkItem()
Traceback (most recent call last):
...
caluma.caluma_workflow.models.WorkItem.task.RelatedObjectDoesNotExist: WorkItem has no task.
```
The question is if people using extension point will use this to create work items, and if we want to support this. But for simplicity's sake I think we can keep the code this way.

Reply:
Of course! Using `pre_save` now.